source/user_guide.rst

```diff
@@ -0,0 +1,67 @@
+User Guide
+==========
+
+The git packaging workflow is currently an opt-in mechanism for submissions to
+openSUSE:Factory via `src.opensuse.org <https://src.opensuse.org>`_ instead of
+`OBS <https://build.opensuse.org/>`_. It allows packagers to maintain their
+package sources in the `pool <https://src.opensuse.org/pool>`_ organization
+instead of on OBS. Submissions are handled via pull requests on
+https://src.opensuse.org instead of submit requests on OBS.
+
+A bot has been setup that will automatically forward eligible pull requests as
+submitrequests directly to openSUSE:Factory.
+
+
+Packager Guide
+--------------
+
+.. hint::
+   Consider adjusting the package description to alert contributors that
+   ordinary submitrequests are no longer possible and where they should send a
+   pull request instead.
+
+To participate in the test drive, modify the development package in the
+developroject as follows:
+
+1. Login to https://src.opensuse.org using the openSUSE IDP provider.
+2. Checkout the latest state of your develop package from OBS. If it differs
```

Comment:
> develop package from OBS

What’s that? `devel/` branch? Personal fork of the package?

Reply:
That should be the package in the develproject. The issue is that the git repository in `pool/` could be outdated and if you change your package in your develproject to scmsync, you'll revert it back to a previous state.

Reply:
Is the current version more understandable?